.mergify.yml

```
@@ -0,0 +1,38 @@
+queue_rules:
+  - name: main
+    batch_size: 3
+    batch_max_wait_time: 4h
+    checks_timeout: 12h
+    merge_method: squash
+    update_method: rebase
+    merge_conditions:
+      - check-success = test-suite-e2e-tests/fhevm-e2e-test (bpr)
+    queue_conditions:
+      - base = main
+      - "#approved-reviews-by >= 1"
+      - label!=do-not-merge
+      - status-success = coprocessor-db-migration-docker-build/build (bpr)
+pull_request_rules:
+  - name: Auto-queue PRs
+    description: Queue a PR when developer comments `@mergify queue`
```

This is actually not true, this will queue the PR automatically if it matches the conditions below + the `queue_conditions`.
The commande `@mergifyio queue` works no matter your pull request rules.
```suggestion
    description: Queue a PR for main when is has been approved
```